src/modules/jcms_rest/src/Plugin/Field/FieldTypeExtend/StringItem.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Drupal\jcms_rest\Plugin\Field\FieldTypeExtend;
+
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\Plugin\Field\FieldType\StringItem as StringItemExtend;
+
+/**
+ * {@inheritdoc}
+ */
+class StringItem extends StringItemExtend {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function generateSampleValue(FieldDefinitionInterface $field_definition) {
+    switch ($field_definition->getName()) {
+      // Random values for orcid need to match the regex ^[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]$.
+      case 'field_person_orcid':
+        $orcid = [4, 4, 4, 3];
+        array_walk($orcid, function (&$value) {
```

Comment:
array_map is usually easier to use as it doesn't modify the original array, but it's functionally the same here

Reply:
Will take a look. Would you prefer to use a library that generated a value based upon the regex?

Reply:
No need, 5 lines of code are very understandable